oc.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <chrono>
```

Comment:
If an interface doesn't depend on these parts of library, they can be safely moved to an implementation file.